styles/globals.css

```diff
@@ -1,10 +1,32 @@
-@import url('https://fonts.googleapis.com/css2?family=Rubik:ital,wght@0,300;0,400;0,500;0,700;1,300;1,500&display=swap');
-@import url('https://fonts.googleapis.com/css2?family=Inconsolata:wght@500&display=swap');
+@import url("https://fonts.googleapis.com/css2?family=Rubik:ital,wght@0,300;0,400;0,500;0,700;1,300;1,500&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=Inconsolata:wght@500&display=swap");
 
 @tailwind base;
 @tailwind components;
 @tailwind utilities;
 
 .w-fitcontent {
     width: fit-content;
-}
\ No newline at end of file
+}
+
+@layer base {
```

Comment:
nonono. i didn't mention this cause i thought you knew. remove this and do something like this in `_app.js`:

```js
import '../styles/globals.css';
import { MDXProvider } from '@mdx-js/react'
import MDXComponents from '../components/MDXComponents'
import { AuthProvider } from '../lib/auth'

function MyApp({ Component, pageProps }) {
  return (
    <AuthProvider>
      <MDXProvider components={MDXComponents}>
          <Component {...pageProps} />
      </MDXProvider>
    </AuthProvider>
  );
}

export default MyApp;
```

and then in `MDXComponents.js`:
```js

const CustomLink = (props) => {
    const href = props.href;
    const isInternalLink = href && (href.startsWith('/') || href.startsWith('#'));

    if (isInternalLink) {
        return (
            <Link href={href}>
                <a {...props} />
            </Link>
        );
    }

    return <a target="_blank" rel="noopener noreferrer" {...props} />;
};

const CustomHeader = (props) => {
    return (
        <h1 className="text-xl font-bold mt-4 mb-1" {...props} />
    )
}

const MDXComponents = {
    h1: CustomHeader,
    a: CustomLink,
};

export default MDXComponents;
```
> By the way, the link component above automatically put `next/link` when it's a client side redirect :)